tests/cli/test_compose_tar_kickstart.sh

```diff
@@ -0,0 +1,133 @@
+#!/bin/bash
+# Note: execute this file from the project root directory
+
+#####
+#
+# Build tar image and install it using liveimg kickstart command
+#
+#####
+
+set -e
+
+. /usr/share/beakerlib/beakerlib.sh
+. $(dirname $0)/lib/lib.sh
+
+CLI="${CLI:-./src/bin/composer-cli}"
+QEMU="qemu-system-$(uname -m)"
+QEMU_CMD="$QEMU -machine accel=kvm:tcg"
+
+rlJournalStart
+    rlPhaseStartSetup
+        rlAssertExists $(which $QEMU)
+        if ! rlCheckRpm httpd; then
+            dnf -y install httpd
+        fi
+        systemctl start httpd
+
+        ks_path="/var/www/html/ks-tar.cfg"
+        tmp_dir=$(mktemp -d /tmp/composer.XXXXX)
+        ssh_key_dir=$(mktemp -d /tmp/composer-ssh-keys.XXXXXX)
+
+        rlRun -t -c "ssh-keygen -t rsa -N '' -f $ssh_key_dir/id_rsa"
+        pub_key=$(cat $ssh_key_dir/id_rsa.pub)
+
+        bp_name="test-tar"
+        blueprint="$bp_name.toml"
+        cat > $blueprint << __EOF__
+name = "$bp_name"
+description = "tar image test"
+version = "0.0.1"
+modules = []
+
+[[packages]]
+name = "openssh-server"
+version = "*"
+
+[[groups]]
+name = "anaconda-tools"
```

Comment:
this shouldn't be needed with the new image type

Reply:
I wasn't 100% sure about this one. But if the reason `@anaconda-tools` group needed to be included was `grub2` and `grub2-tools`, then it can be safely removed from the blueprint.

Reply:
well Brian says the new image type will automatically add everything needed so we don't need to add anything extra on our own. Remove the groups section, tests should continue working.